src/pint/toa.py

```diff
@@ -1675,6 +1677,13 @@ def get_clusters(self, gap_limit=2 * u.h, add_column=False):
             if add_column:
                 self.table.add_column(clusters, name="clusters")
                 self.table.meta["cluster_gap"] = gap_limit
+                log.debug(f"Added 'clusters' column to TOA table with gap={gap_limit}")
+            if add_flag is not None:
```

Comment:
What happens if they pass in True or 1? That seems like a likely error. Probably True will produce `-True 7` and `1` should produce an exception because `-1` is not a valid TOA flag but it's worth checking.

Reply:
It should be the case that if you try to set an invalid flag you get an exception that tells you you did that, which would be a totally reasonable outcome.

Reply:
Er, what I meant was, please try your code (maybe in the test suite!) with some bogus values and check that meaningful exceptions are emitted.

Reply:
OK, I see.  `check_allowed_key` did those checks, and the exceptions are properly raised.